src/KernelIntegrator/RKIntegrationKernel.hh

```diff
@@ -44,7 +44,9 @@ public:
   
   RKIntegrationKernel(const TableKernel<Dimension>& kernel);
   
-  virtual double extent(const Scalar Hmult) const { return mKernel.kernelExtent() / Hmult; }
+  virtual double extent(const Scalar Hmult) const override {
```

Comment:
Adding the override keyword here is great, but I don't care for converting this to a multiline function.  My general style is if an inline method is going to be multiple lines keep it out of the header and put it in the *Inline.hh file.  In this case this could just stay one line with the override added.

Reply:
Fixed.